### DISCUSSION TITLE
An error has occurred. Username or password is incorrect. Try again

### DISCUSSION THREAD

**Discussion by shahram7** [dani-garcia/vaultwarden#3536] [Q&A] [answered]
Hi everyone, 
Yesterday I installed the vaultwarden using a template in portainer. It uses the latest image and default port numbers.

```
		{
			"categories": [
				"Other",
				"Tools"
			],
			"description": "This is a Bitwarden server API implementation written in Rust compatible with upstream Bitwarden clients*, perfect for self-hosted deployment where running the official resource-heavy service might not be ideal..",
			"image": "vaultwarden/server:latest",
			"logo": "https://raw.githubusercontent.com/pi-hosted/pi-hosted/master/images/bitwarden.png",
			"name": "vaultwarden",
			"note": "<b>Template created by Pi-Hosted Series</b><br><b>Check our Github page: <a href=\"https://github.com/pi-hosted/pi-hosted\" target=\"_blank\">https://github.com/pi-hosted/pi-hosted</a></b><br><br><b>Official Webpage: </b><a href=\"https://github.com/dani-garcia/vaultwarden\" target=\"_blank\">https://github.com/dani-garcia/vaultwarden</a><br><b>Official Docker Documentation: </b><a href=\"https://github.com/dani-garcia/vaultwarden/wiki\" target=\"_blank\">https://github.com/dani-garcia/vaultwarden/wiki</a><br><br><br>Alternative Implementation of the Bitwarden Server API in Rust, including web vault.",
			"platform": "linux",
			"ports": [
				"8010:80/tcp",
				"3012:3012/tcp"
			],
			"restart_policy": "unless-stopped",
			"title": "Vaultwarden",
			"type": 1,
			"volumes": [
				{
					"bind": "/portainer/Files/AppData/Config/Bitwarden",
					"container": "/data"
				}
			]
		},

```		
		
in addition I've created a public hostname in my cloudflare zero trust tunnel and so I don't have HTTPS problem anymore. It worked wonderfully until a few hours ago. I think until I've restarted the Proxmox VM (Ubuntu 22.04.2 LTS) , which I use for my Docker containers.
Now I'm not able to login into my account. I'm sure that the master password is correct because I can login in windows app and android app (Not able to sync though) but the web and chrome plugins say
```
 An error has occurred.
Username or password is incorrect. Try again

```
I was really enthusiastic about it, can anybody tell me what's my problem?

**Comment by BlackDex** [comment, marked as answer]
Double check the admin interface, and see if the user exists.
I don't know which type of database you used, and if you are using volatile storage storage, it could mean your database is gone and started again with a fresh one.

**Comment by shahram7** [comment]
@BlackDex thank you very much. 
I'm totally noob and new to docker. I had same problem with other containers. 
I have had docker on a LXC and then I have tried to create a VM for it. What kind of database do you suggest? 
As I deployed Wallabag from compose file, it has created a mariadb container within the stack. Should it be the same with Vaultwarden?
Can I passthrough my second ssd to my docker vm and create a volume on this ssd, in order to use it as a Persistent storage for all of my docker containers?

**Reply by BlackDex** [reply]
On the specifics on how Proxmox works together with LXC/LXD and containers, I have no clue (To long a go that i used Proxmox).
If the deployment tool/script/file you used creates a MariaDB instance, and makes sure it connects to that database, i suggest to connect to that database your self in some way, and see if there is a user in the user table. If not, then something went wrong there.

But, i have no clue on the inner workings of the platform and tools you used.
